mlx/primitives.cpp

```diff
@@ -1,8 +1,8 @@
 // Copyright © 2023 Apple Inc.
-
 #include <algorithm>
 #include <cassert>
 #include <cmath>
+#include <iostream> // TODO
```

Comment:
There is a couple of forgotten `iostream` includes.

Reply:
Just making sure you are paying attention 🤣  

sorry to keep doing this, will be more careful...